### PR TITLE
Change argument dropping to allow missing args in config

### PR DESCRIPTION
**Pull request by ghost**
This was noticed when removing the `mock_args` option entirely
from the config file.

The `None` value would not be dropped and it would break the builds
scripts.

### CODE REVIEW

**Comment by ghost** [comment]
I'd rephrase the commit message to make it clear that we are allowing _empty_ args (`""`), not _missing_ (`None`) args:

Change argument dropping to allow empty args in config

This was noticed when removing the `mock_args` option entirely from the config file.

The empty string (default value for `mock_args`) would be dropped and it would break the builds scripts.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
start build

**Comment by snowcrshd** [comment]
start build

**Comment by snowcrshd** [comment]
test plz
